src/provider/provider_os_memory.c

```diff
@@ -279,14 +356,36 @@ static umf_result_t translate_params(umf_os_memory_provider_params_t *in_params,
     }
     LOG_INFO("established HWLOC NUMA policy: %u", provider->numa_policy);
 
+    result = validate_numa_mode(in_params->numa_mode, emptyNodeset);
+    if (result != UMF_RESULT_SUCCESS) {
+        LOG_ERR("incorrect NUMA mode (%u) or wrong params",
+                in_params->numa_mode);
+        return result;
+    }
```

Comment:
This code is duplicated, I don't see why.

Reply:
Copy-paste error done